cmd/trade.go

```diff
@@ -496,10 +501,44 @@ func convertDeprecatedBotConfigValues(l logger.Logger, botConfig trader.BotConfi
 
 func runTradeCmd(options inputs) {
 	l := logger.MakeBasicLogger()
-	botConfig := readBotConfig(l, options)
+	botStart := time.Now()
+	botConfig := readBotConfig(l, options, botStart)
 	botConfig = convertDeprecatedBotConfigValues(l, botConfig)
 	l.Infof("Trading %s:%s for %s:%s\n", botConfig.AssetCodeA, botConfig.IssuerA, botConfig.AssetCodeB, botConfig.IssuerB)
 
+	// TODO DS Fail if in release mode with undefined API key.
+
+	userID := "-1" // TODO DS Properly generate and save user ID.
```

Comment:
see issue filed for this #516

Reply:
Awesome! Keeping the TODO for organizational purpose.

Reply:
Note that having an ID of -1 results in the HTTP response below. We'll have to prioritize this issue for Amplitude to be functional.

```response: map[code:400 error:Invalid id provided: omit the user_id or device_id field if they are unknown events_with_invalid_ids:map[device_id:[0] user_id:[0]]]```

Reply:
no worries, this will be disabled when checked-in at first, until we are ready to turn it on